python/ray/util/placement_group.py

```diff
@@ -8,8 +8,8 @@ class PlacementGroup:
     """A handle to a placement group.
 
     Args:
-        id: Placement group id.
-        bundles: List of bundles.
+        id (PlacementGroupID): Placement group id.
+        bundles (List): List of bundles.
```

Comment:
BTW @rkooo567 consider explicitly excluding this __init__ in the docs. It's not user facing.